src/util/styles/style-object.js

```diff
@@ -276,7 +276,7 @@ class StyleObject {
 		}
 
 		let selector = selectorCallback ? selectorCallback( getAttribute, attributes ) : _selector
-		let hoverSelector = hoverSelectorCallback ? hoverSelectorCallback( getAttribute, attributes ) : _hoverSelector
+		let hoverSelector = hoverSelectorCallback ? hoverSelectorCallback( getAttribute, attributes, renderIn ) : _hoverSelector
```

Comment:
Let's not add `renderIn` in here. It's better to have another style declaration for `edit` and another for `save` in `style.js` if you need something different rendered.